detox/src/configuration/collectCliConfig.js

```diff
@@ -55,6 +55,7 @@ function collectCliConfig({ argv }) {
     useCustomLogger: asBoolean(get('use-custom-logger')),
     retries: asNumber(get('retries')),
     inspectBrk: asBoolean(get('inspect-brk')),
+    start: get('start'),
```

Comment:
I was actually wondering if it makes sense to add this setting to the behavior config. So that users can disable start commands.


But I decided not to. If this turns out to be a requested feature, we can add it later.